.evergreen/aws_lambda/teardown-atlas-cluster.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Explanation of required environment variables:
+#
+# DRIVERS_ATLAS_PUBLIC_API_KEY: The public Atlas key for the drivers org.
+# DRIVERS_ATLAS_PRIVATE_API_KEY: The private Atlas key for the drivers org.
+# DRIVERS_ATLAS_GROUP_ID: The id of the individual projects under the drivers org, per language.
+# LAMBDA_STACK_NAME: The name of the stack on lambda "dbx-<language>-lambda"
+
+# Explanation of generated variables:
+#
+# FUNCTION_NAME: Uses the stack name plus the current commit sha to create a unique cluster and function.
+# ATLAS_BASE_URL: Where the Atlas API root resides.
+
+# The base Atlas API url. We use the API directly as the CLI does not yet
+# support testing cluster outages.
+ATLAS_BASE_URL="https://cloud.mongodb.com/api/atlas/v1.0"
```

Comment:
Should we decouple the version in the ATLAS_BASE_URL?

```suggestion
ATLAS_API_VERSION="v.10"
ATLAS_BASE_URL="https://cloud.mongodb.com/api/atlas/$ATLAS_API_VERSION"
```

Reply:
Done.